tests/test_transformer.py

```diff
@@ -117,37 +111,25 @@ def test_can_resize_images_with_detection_error_not_ignoring_it(self):
         context = test_data.to_context(detectors=[MockErrorSyncDetector], ignore_detector_error=False)
         trans = Transformer(context)
 
-        trans.transform(None)
+        trans.transform(lambda: None)
 
         expect(test_data.engine.calls['resize']).to_length(0)
 
     def test_can_fit_in(self):
-        for item in FIT_IN_CROP_DATA:
-            context = item[0].to_context()
+        for (test_data, (width, height, should_resize)) in FIT_IN_CROP_DATA:
+            context = test_data.to_context()
             engine = context.modules.engine
 
             trans = Transformer(context)
-            trans.transform(self.validate_fit_in(item, context, engine, trans))
+            trans.transform(lambda: None)
```

Comment:
Reason for this change is that `expect` failure called inside this callback is never surfaced, so test passes.
I made a commit where [make it always fail inside callback ](https://github.com/kkopachev/thumbor/commit/4811ce64ba0c8f33a20ca1dfdf47c4c6f3e2b3b9) and [it was still passing](https://travis-ci.org/kkopachev/thumbor/jobs/286708390)